doc/sphinxext/redirect_from.py

```diff
@@ -119,3 +118,10 @@ def _generate_redirects(app, exception):
             logger.info('making refresh html file: %s redirect to %s', k, v)
             p.parent.mkdir(parents=True, exist_ok=True)
             p.write_text(html, encoding='utf-8')
+
+
+def _clear_redirects(app):
+    domain = app.env.get_domain('redirect_from')
+    if (domain.redirects):
+        domain.redirects.clear()
+        logger.info('clearing cached redirects')
```

Comment:
Unnecessary parentheses. Also, the log message is written to go first.
```suggestion
    if domain.redirects:
        logger.info('clearing cached redirects')
        domain.redirects.clear()
```

Reply:
Implemented, thank you!